lib/common/common.go

```diff
@@ -35,6 +35,12 @@ var (
 )
 
 const (
+	// AppcDockerImageName is the unmodified name this image was originally
+	// referenced by for fetching, e.g. something like "nginx:tag" or
+	// "quay.io/user/image:latest" This is identical in most cases to
+	// 'registryurl/repository:tag' but will differ for the Dockerhub registry or
+	// if the tag was inferred as latest.
+	AppcDockerImageName     = "appc.io/docker/imagename"
```

Comment:
I'm open to whatever renaming you think fits here, as long as it can be used consistently in consumer code and we don't step into existing names. I'm non-native, so some of those may be a bit off, but here are some possibilities: `originalname`, `externalname`, `intialname`, `nativename` or such.